vent/menus/add_options.py

```diff
@@ -50,8 +51,6 @@ def on_ok(self):
         """
         Take the branch, commit, and build selection and add them as plugins
         """
-        self.parentApp.repo_value['versions'] = {}
-        self.parentApp.repo_value['build'] = {}
```

Comment:
I didn't make those. See: https://github.com/CyberReboot/vent/blob/experimental/vent/menus/add_options.py

Looking at it now though, it seems it might be unnecessary. Would you like me to remove?